examples/misc_gltf_model.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Display a Glb/Gltf object with iTowns</title>
+
+        <meta charset="UTF-8" />
+        <link rel="stylesheet" type="text/css" href="css/example.css" />
+        <link rel="stylesheet" type="text/css" href="css/LoadingScreen.css" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/dat-gui/0.7.6/dat.gui.min.js"></script>
+    </head>
+
+    <body>
+        <div id="viewerDiv"></div>
+        <script src="../dist/itowns.js"></script>
+        <script type="text/javascript">
+            var THREE = itowns.THREE;
+            // iTowns namespace defined here
+            var viewerDiv = document.getElementById("viewerDiv");
+
+            var placement = {
+                coord: new itowns.Coordinates(
+                    "EPSG:4326",
+                    4.829361189714537,
+                    45.78511422365657
+                ),
+                tilt: 30,
+                heading: 180,
+                range: 250,
+            };
+
+            var view = new itowns.GlobeView(viewerDiv, placement);
+
+            var orthoSource = new itowns.TMSSource({
+                crs: "EPSG:3857",
+                isInverted: true,
+                format: "image/png",
+                url: "http://osm.oslandia.io/styles/klokantech-basic/${z}/${x}/${y}.png",
+                attribution: {
+                    name: "OpenStreetMap",
+                    url: "http://www.openstreetmap.org/",
+                },
+                tileMatrixSet: "PM",
+            });
+
+            var orthoLayer = new itowns.ColorLayer("Ortho", {
+                source: orthoSource,
+            });
+
+            view.addLayer(orthoLayer);
+
+            // Load a new gltf object
+            itowns.glTFLoader.load(
+                "https://raw.githubusercontent.com/iTowns/iTowns2-sample-data/master/models/lampadaire/scene.gltf ",
+                (gltf) => {
+                    var model = gltf.scene;
+                    model.scale.set(10, 10, 10);
+
+                    // Set coordinates of the GLB
+                    coord = new itowns.Coordinates(
+                        "EPSG:4326",
+                        4.829501189714537,
+                        45.78510982365657,
+                        12
+                    );
+
+                    // Set position and angle of the glb
+                    model.position.copy(coord.as(view.referenceCrs));
```

Comment:
I think you can replace this line by:
```suggestion
                    model.position.add(coord.as(view.referenceCrs));
```
And remove lines 73-80